src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -28,4 +57,16 @@ export class BreadcrumbItemComponent {
 	@HostBinding("class.bx--breadcrumb-item--current") @Input() current = false;
```

Comment:
This can be undefined, so we preserve the old behaviour
```suggestion
	protected _href;
```

Reply:
This note should be with the `protected _href = "";` and not replacing the `@HostBinding`